app/serializers/course_serializer.rb

```diff
@@ -1,8 +1,54 @@
 class CourseSerializer < ActiveModel::Serializer
   # attributes to be serialized
-  attributes :id, :title, :short_title, :hyperlink, :semester_type, :semester_year, :allow_enrollment
-  # model association
+  attributes :id, :title, :short_title, :hyperlink, :semester_type, :semester_year, :allow_enrollment, :permissions
   has_many :sprints
   has_many :topics
   has_many :projects
+
+  def projects
```

Comment:
@mvuajua 

If one remove this section, the permissions are getting rendered right like:
`'project' => ['update' => true, 'delete' => true]`

If one is using this function for serialization, the permissions are false. Somehow is the scope not recognized
Test this with `./spec/requests/projects_spec.rb`